### DISCUSSION TITLE
Transfer ownership of the plugin jodersky/sbt-jni

### DISCUSSION THREAD

**Discussion by jodersky** [sbt/sbt#6604] [General]
Hi everyone! I'm the maintainer of an sbt plugin for JNI compatibility (https://github.com/jodersky/sbt-jni). Recently I have not had much time to continue maintaining it, and doubt that I will in the future. Thankfully, Grigory Pomadchin (@pomadchin) has offered to continue to maintain the project. Currently the repo is hosted under my personal github account, but we were wondering if it would be advisable to move it to the sbt repo, as it appears to contain many plugins already.

**Comment by eed3si9n** [comment]
Hey Jakob. If you could transfer it to eed3si9n, I'd be happy to move it over to sbt repo and create a team with you and Grigory in it.

**Reply by jodersky** [reply]
done, thanks!

**Reply by jodersky** [reply]
btw, how should we manage maven publishing? Should we keep the same groupid, or is there a central sbt groupid that we should use?

**Reply by eed3si9n** [reply]
It's up to you. I own `com.github.sbt`, and I can set up sonatype token for it for automated publishing. If you think you'd be active enough to push out the releases once in a while, you can continue to use the existing organization (group id).

**Reply by eed3si9n** [reply]
Transfer is done - https://github.com/sbt/sbt-jni. You both will have admin rights to the repo.

**Reply by pomadchin** [reply]
Thanks! @eed3si9n I don't know what is preferrable for now, but could you set `com.github.sbt` CI secrets just in case we agree on pushing into that groupid? 

 cc @jodersky

**Reply by jodersky** [reply]
Thanks! I think that using com.github.sbt would indeed be the simplest.

Alternatively, if we wanted to keep the ch.jodersky group, then we would need to
1) either create a new sonatype user account or use yours, @pomadchin 
2) create a JIRA ticket to ask that that account be given publish rights to ch.jodersky

(unfortunately I can't simply give you the access token to my maven account, as I have access to too many group ids and never separated them into separate accounts)

**Reply by eed3si9n** [reply]
Credentials are set up.

**Reply by pomadchin** [reply]
@eed3si9n thank you, I checked them on snapshot releases and it works 💯
